addon/components/power-calendar-multiple.js

```diff
@@ -30,25 +31,27 @@ export default CalendarComponent.extend({
   actions: {
     select(day, calendar, e) {
       let action = this.get("onSelect");
+      let selected = this.get("publicAPI.selected");
+      
       if (action) {
-        action(this._buildCollection(day), calendar, e);
+        action(this._buildCollection({ date: selected }, day), calendar, e);
       }
     }
   },
 
   // Methods
-  _buildCollection(day) {
```

Comment:
Why did you need to change this?

Reply:
Because in the months branch this function is used in a reduction over all the months in a quarter when quarters are selected.  Selected needs to be explicitly passed because that is where the result is accumulated.

https://github.com/cibernox/ember-power-calendar/pull/173/files#diff-9767bdfed1fc393367efc2fd76225736R45

Reply:
Nothing is stopping this from being reverted in the months branch.  If you want I'll change it back, but of course if both months and years get merged it won't matter.